packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
-import styles from '@patternfly/patternfly-next/components/Avatar/avatar.css';
+import * as React from 'react';
```

Comment:
For this you can set the following in tsconfig.json
```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    ...
  }
}
```

Reply:
@dmiller9911 This worked for style and PropTypes but I get no default error for React so I had to still do * as React.  Thoughts?

Reply:
Got it working